MIGRATING.md

```diff
@@ -66,7 +66,7 @@ As part of this upgrade, we will remove it from the method signatures.
    - Removed `getAllGroupsValid` and `setAllGroupsValid` (property `allGroupsValid` is being removed; 
    this option would be removed from future versions of Okta API)
    - Removed `getEmbedded` (property `_embedded` is removed; this property is undocumented in Okta API and hence being removed)
-   - `delete(Boolean removeUsers)` Signature changed to `delete()`
+   - delete(Boolean removeUsers) Signature changed to delete(); optional parameter `removeUsers` which defaulted to `false` earlier is now removed. 
```

Comment:
Callers who used `removeUsers` value `true` will be impacted.

Reply:
Chat with the team that owns this API, the default may have been changed (or maybe it always removes the users now?)

Reply:
Sorry this one got buried and I lost track.

So, the user will not removed when the group rule is removed.